### PR TITLE
Disable expensive LNC check.

### PR DESCRIPTION
**Pull request by afck**

### CODE REVIEW

**Review by goral09 — APPROVED** [review]

**Review by fizyk20 — APPROVED** [review]

File: node/src/components/consensus/highway_core/state.rs [hunk 1]
```diff
@@ -52,6 +52,9 @@ use tallies::Tallies;
 // endorsements again.
 pub(super) const TODO_ENDORSEMENT_EVIDENCE_DISABLED: bool = true;
 
+// Disables checking the limited naivete criterion for endorsements.
```
Comment (GuybrushX) [inline node/src/components/consensus/highway_core/state.rs:55]: Typo or what is "naivete criterion"?

Comment (GuybrushX) [reply node/src/components/consensus/highway_core/state.rs:55]: I checked the Highway protocol PDF and it should be "naivetY" I guess.

Comment (afck) [reply node/src/components/consensus/highway_core/state.rs:55]: You're right, we should make it consistent. My English dictionary has both "naivety" and "naïveté", though.

**Comment by afck** [comment]
bors merge

**Comment by afck** [comment]
bors merge

**Comment by afck** [comment]
bors merge

**Pull request merged**